processor/src/host/advice/map.rs

```diff
@@ -0,0 +1,43 @@
+use super::{BTreeMap, Felt, Vec};
+
+extern crate alloc;
+use alloc::collections::btree_map::IntoIter;
```

Comment:
Instead of these, we can use `use vm_core::utils::collections::btree_map::IntoIter;`